package.json

```diff
@@ -5,8 +5,7 @@
   "main": "src/index.ts",
   "exports": {
     ".": "./src/index.ts",
-    "./config": "./src/vitepress/config/baseConfig.js",
-    "./highlight": "./src/vitepress/config/highlight.js"
```

Comment:
Seems the highlight export would also be reused here:
https://github.com/vuejs/docs/blob/main/src/tutorial/tutorial.data.ts#L4

Reply:
There too, that `highlight` option can be removed. The default highlighter is (nearly) same as that custom one.